docs/ha-considerations.md

```diff
@@ -83,23 +83,32 @@ There are some placeholders in `bash` variable style to fill in:
 
 The above `keepalived` configuration uses a health check script `/etc/keepalived/check_apiserver.sh` responsible for making sure that on the node holding the virtual IP the API Server is available. This script could look like this:
 
-```
+```bash
 #!/bin/sh
 
 errorExit() {
     echo "*** $*" 1>&2
     exit 1
 }
 
-curl --silent --max-time 2 --insecure https://localhost:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://localhost:${APISERVER_DEST_PORT}/"
-if ip addr | grep -q ${APISERVER_VIP}; then
-    curl --silent --max-time 2 --insecure https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/ -o /dev/null || errorExit "Error GET https://${APISERVER_VIP}:${APISERVER_DEST_PORT}/"
+# replace this with the virtual IP address negotiated between the `keepalived` cluster hosts
+APISERVER_VIP=${APISERVER_VIP_VALUE}
+# replace this with the port through which Kubernetes will talk to the API Server
+APISERVER_DEST_PORT=${APISERVER_DEST_PORT_VALUE}
+
+url="https://localhost:$APISERVER_DEST_PORT"
+if ip addr | grep -q $APISERVER_VIP; then
+        url="https://$APISERVER_VIP:$APISERVER_DEST_PORT"
 fi
+http_code=$(curl --output /dev/null --silent --write-out "%{http_code}" --insecure --max-time 2 ${url})
+err_code=$?
+[ $err_code -eq 0 ] || errorExit "Error get $url, error code $err_code"
+[ "$http_code" = "200" ] || [ "$http_code" = "403" ] || errorExit "Error get $url, status code $http_code"
```

Comment:
echo to the comment above, comment here to explain why "403" is not considered as an error code.

Reply:
@chendave regarding the placeholders: in my original example script there were no variables for `APISERVER_VIP` and `APISERVER_DEST_PORT`, hence these names were placeholders to be filled in when creating the script for a particular network. 

@wb14123 has changed the script a bit, so that now actual variables are used. Hence the above names were used for variables in the script and new names, `APISERVER_VIP_VALUE` and `APISERVER_DEST_PORT_VALUE` were introduced as placeholders instead. Again, these are not meant as "real" variables but values to fill in when creating the script for a network. 

I hope this is understandable now?